lib/lockstep_sdk/clients/activities.rb

```diff
@@ -0,0 +1,110 @@
+#
+# Lockstep Software Development Kit for Ruby
+#
+# (c) 2021-2022 Lockstep, Inc.
+#
+# For the full copyright and license information, please view the LICENSE
+# file that was distributed with this source code.
+#
+# @author     Ted Spence <tspence@lockstep.io>
+# @copyright  2021-2022 Lockstep, Inc.
+# @version    2022.3
+# @link       https://github.com/Lockstep-Network/lockstep-sdk-ruby
+#
+
+
+class activities_client
+
+    # Initialize the activities_client class with a lockstepsdk instance.
+    def initialize(lockstepsdk)
+        @lockstepsdk = lockstepsdk
+    end
+
+
+    # Retrieves the Activity specified by this unique identifier, optionally including nested data sets.
+    # 
+    # An Activity contains information about work being done on a specific accounting task. You can use Activities to track information about who has been assigned a specific task, the current status of the task, the name and description given for the particular task, the priority of the task, and any amounts collected, paid, or credited for the task.
+    # 
+    # @param id [uuid] The unique Lockstep Platform ID number of this Activity
+    # @param include_param [string] To fetch additional data on this object, specify the list of elements to retrieve. Available collections: Company, Attachments, CustomFields, Notes, References, and UserAssignedToName
+    def retrieve_activity(id:, include_param:)
```

Comment:
think instead of `include_param` we could just  rename it to `params`, here and all places should work I guess

Reply:
The API parameter is called "include", but I can't use the name "include" here because that's a Ruby keyword.  I wanted to add something to the name to make it not a keyword - would there be something better?